File: app/src/main/java/com/antonio/samir/meteoritelandingsspots/service/repository/database/AppDatabase.java
package com.antonio.samir.meteoritelandingsspots.service.repository.database;

import com.antonio.samir.meteoritelandingsspots.model.Meteorite;

import androidx.room.Database;
import androidx.room.RoomDatabase;

@Database(entities = {Meteorite.class}, version = 2)
public abstract class AppDatabase extends RoomDatabase {

    public abstract MeteoriteDao meteoriteDao();

}

File: app/src/main/java/com/antonio/samir/meteoritelandingsspots/service/repository/database/AppDataBase.kt
package com.antonio.samir.meteoritelandingsspots.service.repository.database

import androidx.room.Database
import androidx.room.RoomDatabase
import com.antonio.samir.meteoritelandingsspots.model.Meteorite

@Database(entities = arrayOf(Meteorite::class), version = 2)
abstract class AppDataBase : RoomDatabase() {

    abstract fun meteoriteDao(): MeteoriteDao

}
